src/main/java/graphql/ProfilerResult.java

```diff
@@ -273,12 +273,12 @@ public List<String> getInstrumentationClasses() {
 
     public Map<String, Object> shortSummaryMap() {
         Map<String, Object> result = new LinkedHashMap<>();
-        result.put("executionId", Assert.assertNotNull(executionId));
+        result.put("executionId", Assert.assertNotNull(executionId).toString());
         result.put("operation", operationType + ":" + operationName);
         result.put("startTime", startTime);
         result.put("endTime", endTime);
-        result.put("totalRunTime", (endTime - startTime) + "(" + (endTime - startTime) / 1_000_000 + "ms)");
-        result.put("engineTotalRunningTime", engineTotalRunningTime + "(" + engineTotalRunningTime / 1_000_000 + "ms)");
```

Comment:
Change to nanoseconds only, if need be someone can convert to ms later

Remove string and make it a number instead